px-data-grid-column.html

```diff
@@ -121,6 +121,44 @@
             obj.element = this;
           }
         }
+
+        _orderChanged(order, headerCell, footerCell, cells) {
+          if (this.mappedObject && this.mappedObject.order != order) {
```

Comment:
DRY refactoring needed, lines 126-160 looks almost the same as 134-138 and 142-146